rad-gate.ino

```diff
@@ -83,20 +80,23 @@ void setup() {
   //pinMode(PIN_REED_SWITCH, INPUT_PULLUP);
   pinMode(PIN_LIGHT_TREE_RELAY_ENABLE, INPUT_PULLUP);
 
-  // not working? not needed?
+  // Use interrupt to catch ABORT request from GO button
   attachInterrupt(digitalPinToInterrupt(PIN_BUTTON_GO), Interrupt1, HIGH);
-  digitalWrite(PIN_RELAY, HIGH); // turn on magnet
+
+  // turn off solenoid
+  // serial_print("Gate solenoid DEACTIVATED");
+  digitalWrite(PIN_RELAY, LOW);
 
   if (digitalRead(PIN_LIGHT_TREE_RELAY_ENABLE) == LOW) {
     lighttree.initialise(true, PIN_LIGHT_TREE_RELAY_ENABLE);
-    serial_print("Light tree configured for relay control");
-  } else {
+    // serial_print("Light tree configured for relay control");
+  }  else {
     lighttree.initialise(false, PIN_NEO_PIXEL);
     serial_print("Light tree configured for serial control");
   }
 
   serial_print("Gate controller initialised");
-  serial_print(VERSION);
+  serial_print("Waiting for GO Button Press");
```

Comment:
We should keep the version string print, but also add your line. Otherwise it isn't clear we are ready for a button press. Good idea.